test-a11y/config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  toleranceThreshold: 5,
+  toleranceThreshold: 8,
```

Comment:
why is this going up?

Reply:
I added the contrast check back to verify the color contrast is good after these changes and left it on. It's reporting on a few workspace-specific contrast issues. I talked to @seanforyou23 and we're going to work on that when we start work on https://github.com/patternfly/patternfly-next/issues/1602